pkg/dcgmexporter/gpu_collector.go

```diff
@@ -90,9 +90,8 @@ func (c *DCGMCollector) Cleanup() {
 
 func (c *DCGMCollector) GetMetrics() (map[Counter][]Metric, error) {
 	monitoringInfo := GetMonitoredEntities(c.SysInfo)
-	count := len(monitoringInfo)
 
-	metrics := make(map[Counter][]Metric, count)
+	metrics := make(map[Counter][]Metric)
```

Comment:
Just an optional suggestion - may be declare `type GPUMetrics map[Counter][]Metric` in `pkg/dcgmexporter/types.go`, declaring the type would improve readability and optionally provide an opportunity to define methods in future if needed.

Reply:
I will make this change in another PR.